main.tf

```diff
@@ -280,6 +280,14 @@ resource "aws_lambda_event_source_mapping" "this" {
     }
   }
 
+  dynamic "scaling_config" {
+    for_each = try(each.value.maximum_concurrency_sqs_scaling, null) != null ? [true] : []
+    content {
+      maximum_concurrency = maximum_concurrency_sqs_scaling.value
```

Comment:
```suggestion
      maximum_concurrency = scaling_config.value.maximum_concurrency
```

Reply:
resolved in https://github.com/terraform-aws-modules/terraform-aws-lambda/pull/402/commits/748c6c2d8a1152e020fd84ff3ed61d41d117314a